kintone/js-sdk
# feat(rest-api-client): add new properties to get/updateAppSettings

## Why



- Add property for get/update app settings

## What



- Add a document for the new method.
- Add new unit tests.

## How to test

1. run below code and check logs:
```ts

// packages/rest-api-client/test.ts

import { KintoneRestAPIClient } from "./src";

const client = new KintoneRestAPIClient({
  baseUrl: "http://localhost",
  auth: {
    username: "Administrator",
    password: "cybozu",
  },
});

(async () => {
  // アプリを作成
  const respApp = await client.app.addApp({ name: "test app" });
  // アプリ設定を取得
  const respSettings1 = await client.app.getAppSettings({
    app: respApp.app,
    preview: true,
  });
  console.log(respSettings1);
  // アプリ設定を更新
  await client.app.updateAppSettings({
    app: respApp.app,
    name: "new name",
    description: "new description",
    titleField: {
      selectionMode: "AUTO",
    },
    enableThumbnails: false,
    enableComments: false,
  });
  // アプリ設定を取得
  const respSettings2 = await client.app.getAppSettings({
    app: respApp.app,
    preview: true,
  });
  console.log(respSettings2);
})();

```

## Checklist
